NEWS.md

```diff
@@ -1,7 +1,7 @@
 # epinowcast 0.2.0
 
 ## Model
-- Added support for parametric log-logistic delay distributions. See [#128](https://github.com/epiforecasts/epinowcast/pull/128) by [@adrian-lison](https://github.com/adrian-lison).
+- Added support for parametric log-logistic delay distributions. See #128 by @adrian-lison.
```

Comment:
Missing a news entry for this PR. Ideally, we need a comment saying what you have done and then linking to the PR and PR author (as done by Adrian here). 

As this is a package change can you add a ## Package subheading for your new entry as we did for `epinowcast 0.1.0`

Reply:
I have done this